workflows_core/dataset/dataset.py

```diff
@@ -231,3 +231,42 @@ def facets(
             page_size=page_size,
             asc=asc,
         )
+
+    def get_keyphrase(self, field: str, alias: str, keyphrase_id: str):
+        return self._api._get_keyphrase(
+            dataset_id=self.dataset_id,
+            field=field,
+            alias=alias,
+            keyphrase_id=keyphrase_id,
+        )
+
+    def update_keyphrase(self, field: str, alias: str, keyphrase_id: str, update: dict):
```

Comment:
`update: Dict[?: ?]`

Reply:
it can be anything for now - we haven't finalised the schema for this yet.